server/src/app/modules/product/product.service.ts

```
@@ -35,10 +36,38 @@
     return deletedProduct;
 };
 
-const findAllProducts = async () => {
-    return await ProductModel.find().sort({ createdAt: -1 });
-}
+const findAllProducts = async (categoryName: string | undefined, sortBy: string, page: number, limit: number) => {
+    const skip = (page - 1) * limit;
+
+    let sortOption: any = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'sortOption' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)